### DISCUSSION TITLE
backup fails with same bitmap allready exists error again and again

### DISCUSSION THREAD

**Discussion by kmorber** [abbbi/virtnbdbackup#195] [Q&A / Support]
Interner Fehler: Kann Qemu-Befehl 'transaction' nicht ausführen: Bitmap already exists

backup fails again and again at virtnbdbackup.14. when i reset the bitmaps, all backups are working until virtnbdbackup.14.

**Comment by abbbi** [comment]
please supply logs for all backups prior to that point
without loggings, possible error cause cannot be determined. I havent had any user report this issue before and i have done by far more than 14 backups. There is no code involved that limits the amount of bitmaps. This must be either caused by a third party component or is caused by another specific thing happening on your system.

Please also check the libvirt/qemu logfiles for what happened between backup 13 and backup 14 to the virtual machine (migrated to another host? power state changes? etc etc).

Im converting this into an discussion because its by no means clear this is a bug or an issue in virtnbdbackup.

**Comment by abbbi** [comment]
see also:   https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backup-fails-with-unable-to-execute-qemu-command-transaction-bitmap-already-exists

**Comment by abbbi** [comment]
hi, can you share what the cause was, thanks
